recipe/meta.yaml

```
@@ -9,8 +9,8 @@ package:
   version: {{ version }}
 
 source:
-  url: https://github.com/openmm/openmm-torch/archive/refs/tags/v{{ version }}.tar.gz
-  sha256: f11a2a1f58aa4a58d6c9355f7248a36043966771cb1be028a6f847c258ff4786
+  git_url: https://github.com/openmm/{{ name }}.git
```

```suggestion
  git_url: https://github.com/openmm-torch/{{ name }}.git
```

Shouldn't this be `openmm-torch`?

Also, you can get tarballs from a specific commit too:

```
  url: https://github.com/openmm/openmm-torch/archive/{{ commit }}.tar.gz
```

---

Wrong URL!

---

Oops, thanks!